src/Enums.js

```diff
@@ -11,7 +11,9 @@ exports.UmbrellaId = {
 exports.StringFormat = {
     DATE: 'YYYY-MM-DD', // '2018-03-24'
     TIME: 'HH:mm', // '15:30' (24hr),
-    PHONE: '[0-9]{3}-[0-9]{3}-[0-9]{4}' // '206-345-7890'
+    PHONE: '[0-9]{3}-[0-9]{3}-[0-9]{4}', // '206-345-7890',
+    WEEKDAY: 'dddd', // 'Wednesday'
+    WEEKDAY_WITH_DATE: 'dddd, YYYY-MM-DD' // 'Wednesday, 2018-04-25'
```

Comment:
@arielin3 I left this as YYYY-MM-DD for consistency with the existing datestring format, but I think we should do MM/DD/YYYY unless there was some other reason we were doing this. Also, can we rename this from `StringFormat` to `DateStringFormat`?

Reply:
@jkbach Ah sorry, the reason it was YYYY-MM-DD is because this enum was more used for backend purposes 😂It was so that we can make sure we write/read things in the same format. But now that we are going to store everything in timestamps, this kinda changed into display formats.

And I changed it to `StringFormat` from `DateTimeFormat` because i needed to add the format for `phone`. Feel free to change it to a better name, or suggest a different structure? Do we want to maybe have separate enums?